### ISSUE TITLE
Feature Request Tab and arrow up down to highlight. Enter to select.

### ISSUE THREAD

**Issue by kenthinson** [felixlucien/flutter-autocomplete-textfield#62]
This is a very cool widget. However flutter web and desktop are coming. As well as iPad with keyboard attached. It would be great if this widget supported keyboard interactions. Like the normal up / down arrow to highlight or tab / shift tab and enter to select.

**Comment by mmtolsma** [comment]
Agreed. I am also using this for an android tablet application and the use of arrows or tab keys to navigate would be amazing.
